src/styles/global.css

```diff
@@ -31,3 +31,18 @@
   --color-accent-900: var(--color-cerulean-blue-900);
   --color-accent-950: var(--color-cerulean-blue-950);
 }
+
+figure:not(.is-terminal) code {
+  counter-reset: step;
+  counter-increment: step 0;
+}
+
+figure:not(.is-terminal) code .ec-line::before {
+  content: counter(step);
+  counter-increment: step;
+  width: 2rem;
+  margin-right: 0rem;
+  display: inline-block;
+  text-align: right;
+  color: rgba(115, 138, 148, 0.4);
+}
```

Comment:
[minor]: Will it be valuable for the docs styles? if not, consider moving it to a separate file `blog.css` that we can include into blog pages only.